react-app/src/themes/appTheme.ts

```diff
@@ -1,5 +1,6 @@
 import { createTheme } from '@mui/material';
 import '@bcgov/bc-sans/css/BC_Sans.css';
+import type {} from '@mui/x-data-grid/themeAugmentation';
```

Comment:
I thought this was odd, but it's official documentation. 🤷

Reply:
Yeah, not really sure how this one works exactly, but that's what it says...